lib/engine/game/g_1848/game.rb

```diff
@@ -966,6 +966,15 @@ def first_column?(entity)
           _r, c = entity.share_price.coordinates
           c == 1
         end
+
+        def next_round!
+          reset_company_values if @round.is_a?(Engine::Round::Auction)
```

Comment:
I think this will be called if everyone passes. Please verify.

Reply:
I verified, it doesn't. If everyone passes the auction round is still active.
<img width="466" alt="image" src="https://user-images.githubusercontent.com/108803147/185045861-3f275fa3-4767-4977-8d7d-a2d549585cf6.png">
 and 
 
<img width="515" alt="image" src="https://user-images.githubusercontent.com/108803147/185045889-f2373b47-f415-4631-a4f2-dcb5f231487b.png">